src/porepy/numerics/parabolic.py

```diff
@@ -103,7 +115,11 @@ def step(self):
     def update(self, t):
         'Update parameters to time t'
         for g, d in self.grid():
-            d['problem'].update(t)
+            d[self.physics + '_data'].update(t)
+
```

Comment:
The data assigner should now be stored with keyword, e.g., 'flow_data'. I think this is quite consistent with the physics consept used in the parameter class

Reply:
Good.